### PR TITLE
Fix: changed signatory name source priority

### PR DESCRIPTION
**Pull request by egor0798**
Fixes:

- Changed signatory name source priority
- Added padding for version block on settings

### CODE REVIEW

**Review by Asmadek — APPROVED** [review]

File: src/renderer/screens/Operations/common/utils.ts [hunk 1]
```diff
@@ -219,3 +222,23 @@ export const getTransactionAmount = (tx: Transaction): string | null => {
 
   return null;
 };
+
+export const getSignatoryName = (
```
Comment (tuul-wq) [inline src/renderer/screens/Operations/common/utils.ts:226]: Better use domain types, there are no `id's` down below
```suggestion
export const getSignatoryName = (
  signatoryId: AccountId,
  tx: MultisigTransaction,
  contacts: Contact[],
  accounts: Account[],
  addressPrefix?: number,
): string => {
```

File: src/renderer/screens/Operations/common/utils.ts [hunk 2]
```diff
@@ -219,3 +222,23 @@ export const getTransactionAmount = (tx: Transaction): string | null => {
...
+  signatoryId: AccountId,
+  tx: MultisigTransaction,
```
Comment (tuul-wq) [inline src/renderer/screens/Operations/common/utils.ts:228]: If we only get `signatories` from `tx` then it's preferably to pass `tx.signatories`
So the argument would become - `signatories: Signatory[]`

File: src/renderer/screens/Operations/components/Log.tsx [hunk 3]
```diff
@@ -37,50 +34,29 @@ const EventMessage: Partial<Record<SigningStatus | 'INITIATED', string>> = {
...
-  };
-
   const getEventMessage = (event: MultisigEvent): string => {
-    const signatory = getSignatory(event.accountId);
     const isCreatedEvent =
-      signatory?.accountId === tx.depositor && (event.status === 'SIGNED' || event.status === 'PENDING_SIGNED');
+      event.accountId === tx.depositor && (event.status === 'SIGNED' || event.status === 'PENDING_SIGNED');
 
-    const signatoryName = signatory?.name || toAddress(event.accountId, { chunk: 5, prefix: chain?.addressPrefix });
+    const signatory = getSignatoryName(event.accountId, tx, contacts, accounts, connection?.addressPrefix);
```
Comment (tuul-wq) [inline src/renderer/screens/Operations/components/Log.tsx:55]: `signatoryName` was a good name, even function are called so ☺️

File: src/renderer/screens/Operations/components/Log.tsx [hunk 4]
```diff
@@ -1,30 +1,27 @@
...
-import { AccountId } from '@renderer/domain/shared-kernel';
-import { useContact } from '@renderer/services/contact/contactService';
-import { useAccount } from '@renderer/services/account/accountService';
+import { AccountDS, ContactDS } from '@renderer/services/storage';
 
 type Props = {
   tx: MultisigTransaction;
   account?: MultisigAccount;
   connection?: ExtendedChain;
+  accounts: AccountDS[];
```
Comment (tuul-wq) [inline src/renderer/screens/Operations/components/Log.tsx:23]: `Account` `Contact` if possible 👍

File: src/renderer/screens/Operations/components/OperationFullInfo.tsx [hunk 5]
```diff
@@ -144,14 +152,15 @@ const OperationFullInfo = ({ tx, account }: Props) => {
         </div>
 
         <ul className="flex flex-col gap-y-0.5">
-          {signatoriesList.map(({ accountId, name }) => {
+          {signatoriesList.map(({ accountId, matrixId }) => {
             return (
```
Comment (tuul-wq) [inline src/renderer/screens/Operations/components/OperationFullInfo.tsx:156]: `return` can be omitted 🙂

File: src/renderer/components/common/SignatoryCard/SignatoryCard.tsx [hunk 6]
```diff
@@ -26,13 +27,21 @@ const SignatoryCard = ({
   addressFont = 'text-body text-inherit',
   size = 20,
   name,
+  matrixId,
   ...addressProps
 }: Props) => {
   const { getLiveAccounts } = useAccount();
   const address = getAddress(addressProps);
   const account = getLiveAccounts().find((a) => toAddress(a.accountId) === address);
```
Comment (tuul-wq) [inline src/renderer/components/common/SignatoryCard/SignatoryCard.tsx:35]: In worst case we're guaranteed to receive `address`
`name={getSignatoryName(accountId, tx, contacts, accounts, connection?.addressPrefix)}`
So condition will never fall into `account?.name` - `<AccountAddress ... name={name || account?.name} />`
Guess it's safe to remove `useAccount` from `SignatoryCard`
Please correct me if I'm wrong 🙏

**Review by tuul-wq — COMMENTED** [review]
Cool!

**Pull request merged**